seatunnel-core/seatunnel-flink-starter/src/main/java/org/apache/seatunnel/core/starter/flink/execution/FlinkExecution.java

```diff
@@ -70,4 +84,32 @@ public void execute() throws TaskExecuteException {
             throw new TaskExecuteException("Execute Flink job error", e);
         }
     }
+
+    /**
+     * return plugin's dependent jars, which located in 'plugins/${pluginName}/lib/*'.
+     */
+    private List<URL> getPluginsJarDependencies() {
```

Comment:
This method is implemented once in both sparkstarter and flinkstarter, and there is almost no difference in the code, so you can put this method as a static method in this class `org.apache.seatunnel.common.config`, 
The method is defined as follows:

```java

List<Path> getPluginsJarDependencies() throws IOException {}

```

In flinkstarter, you can use it like this:

```java

List<URL> pluginDependencies = Common.getPluginsJarDependencies().stream().map(jar -> jar.toUri().toURL())

```

This can greatly reduce the redundancy of the code, what do you think about it?